doc/release/0.18.0-notes.rst

```diff
@@ -65,6 +65,9 @@ is now consistently added after the matrix is applied,
 independent of if the matrix is specified using a one-dimensional
 or a two-dimensional array.
 
+``stats.ks_2samp`` now only accepts real, non-nan inputs. It used to return
+nonsensical values for such inputs before.
```

Comment:
It used to return nonsensical values for ... inputs which are not real and not nan? I'm just confused what "such" stands for here.